charts/cluster-api-provider-aws/templates/apps_v1_deployment_capa-controller-manager.yaml

```diff
@@ -36,6 +36,7 @@ spec:
       containers:
       - args:
         - --leader-elect
+        - --disable-controllers=awsmachine,awscluster
```

Comment:
I don't think this is the right flag check the proposal [here](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/docs/proposal/20250314-optional-controllers.md) and the constant value is [unmanaged](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/controllers/disabled_controllers.go#L27) . Would you test make sure that the flag is disabling the controllers

Reply:
yeah, you are right. I have updated the code and verified that CAPA starts with `awscluster` and `awsmachine` controllers are turned off.
```
I0519 10:57:13.218307       1 logger.go:78] "controller disabled" logger="setup" controller="AWSMachine" controller-group="unmanaged"
I0519 10:57:13.218316       1 logger.go:78] "controller disabled" logger="setup" controller="AWSCluster" controller-group="unmanaged"
```